packages/admin-panel/src/dataTables/query/useDataTablePreview.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { useQuery } from 'react-query';
+import { post } from '../../VizBuilderApp/api/api';
+import { DEFAULT_REACT_QUERY_OPTIONS } from '../../VizBuilderApp/api/constants';
+
+/**
+ * {
+    "code": "data_lake_db_test",
+    "type": "sql",
+    "config": {
+      "sql": "SELECT * FROM Table",
+      "externalDatabaseConnectionCode": "db_code",
+      "additionalParameters": [
+        {
+          "name": "entityCode",
+          "config": {
+            "type": "string"
+          },
+          "inputFilterValue": "AU_SA%"
+        }
+      ]
+    },
+    "permission_groups": [
+      "*"
+    ]
+  }
+
+* => 
+  * {
+    "code": "data_lake_db_test",
+    "type": "sql",
+    "config": {
+      "sql": "SELECT * FROM Table",
+      "externalDatabaseConnectionCode": "db_code"
+    },
+    "additionalParameters": [
+      {
+        "name": "entityCode",
+        "config": {
+          "type": "string"
+        }
+      }
+    ],
+    "entityCode": "AU_SA%",
+    "permission_groups": [
+      "*"
+    ]
+  }
+*/
+const pullValidConfig = previewConfig => {
```

Comment:
I'm assuming this is a difference between the frontend and the backend? If so one of these should change so there is consistency 😄